.ci/run_tests.sh

```diff
@@ -4,11 +4,25 @@ set -e
 function cyan() {
     echo -e "\033[1;36m$*\033[0m"
 }
+
 function red() {
     echo -e "\033[1;31m$*\033[0m"
 }
 
-export BUSTED_ARGS="--no-k -o htest -v --exclude-tags=flaky,ipv6"
+function get_failed {
+    if [ ! -z "$FAILED_TEST_FILES_FILE" -a -f "$FAILED_TEST_FILES_FILE" ]
+    then
+        sort -u < $FAILED_TEST_FILES_FILE
+    else
+        echo "$@"
```

Comment:
Why do we sort the file when rerun but not at the first time?

Reply:
I think the sort is not really needed, so I've removed it.